install/jaxrs/bin/ts.jte

```diff
@@ -77,20 +75,18 @@ javatest.timeout.factor=1
 ###################################################################
 ###################################################################
 ###################################################################
-##  CI SPECIFIC PROPERTIES LIVE BELOW
+##  RI SPECIFIC PROPERTIES LIVE BELOW
```

Comment:
Please change back to CI instead of RI.

Reply:
For making JDK 11 as base, I have replaced ts.jte with ts.jte.jdk11. issue is from https://github.com/eclipse-ee4j/jakartaee-tck/blob/master/install/jaxrs/bin/ts.jte.jdk11, rectified same.